packages/sentinel_one_cloud_funnel/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.3.2"
```

```suggestion
- version: "1.4.0"
```
(enhancements bump minor)